packages/web/docs/src/content/management/targets.mdx

```diff
@@ -214,3 +184,37 @@ in the list of CDN tokens.
 <Callout type="info">
   Revoking CDN access token can take up to 5 minutes to update across the CDN network.
 </Callout>
+
+### Registry Access Tokens
+
+<Callout type="warning">
+
+Please use [Access Tokens](/docs/management/access-tokens), which allow more granular access
+control, instead of Registry Access Tokens. Registry Access Tokens are deprecated and will be
```

Comment:
here again i dont love the terminology. Maybe "Granular Access Control" instead of just "Access Tokens"?